datalad/distribution/publish.py

```diff
@@ -255,7 +256,10 @@ class Publish(Interface):
         recursion_limit=recursion_limit,
         git_opts=git_opts,
         annex_opts=annex_opts,
-        annex_copy_opts=annex_copy_opts
+        annex_copy_opts=annex_copy_opts,
+        # "interferes" with skip_failing which is actually "skip_missing"
+        # TODO: unify as --missing=(skip|inherit) ?
```

Comment:
ATM it's `skip_missing` indeed. But `skip_failing` is also used, when there was no sibling specified and we can not figure out one automatically. In that case `inherit` wouldn't work, since we don't know what to inherit from. With proper documentation, `inherit` could then fall back to `skip` behavior, but we then also need a third value `fail`, I guess.